pkg/setting/setting_navigation.go

```diff
@@ -0,0 +1,24 @@
+package setting
+
+import (
+	"strings"
+)
+
+func (cfg *Cfg) readNavigationSettings() error {
+	sec := cfg.Raw.Section("navigation")
+	cfg.PluginAppNavIds = map[string]string{
```

Comment:
Should this live in the `default.ini`? Maybe I'm not reading the code correctly but it looks like right now these monitoring and alerts-and-incidents sections will be enabled for any `navigation` config.

Also should we update `sample.ini` in this PR?

Reply:
yea, they could be there but was thinking it made better sense to just specify them here. 

> Maybe I'm not reading the code correctly but it looks like right now these monitoring and alerts-and-incidents sections will be enabled for any navigation config.

These are only used when topnav is enabled